### PR TITLE
support logLoss in transform evaluate for reduced copy cv

### PR DESCRIPTION
**Pull request by eordentlich**

### CODE REVIEW

**Comment by eordentlich** [comment]
build

File: python/src/spark_rapids_ml/metrics/__init__.py [hunk 1]
```diff
@@ -13,3 +13,28 @@
...
+# Global parameter used by core and subclasses.
+TransformEvaluateMetric = namedtuple(
+    "TransformEvaluateMetric", ("accuracy_like", "log_loss", "regression")
+)
+transform_evaluate_metric = TransformEvaluateMetric(
+    "accuracy_like", "log_loss", "regression"
+)
+
+
+@dataclass
```
Comment (wbo4958) [inline python/src/spark_rapids_ml/metrics/__init__.py:30]: First time knew it.

File: python/src/spark_rapids_ml/classification.py [hunk 2]
```diff
@@ -507,50 +537,89 @@ def _transformEvaluate(
...
+                    label_count=label_count[i],
+                )
+                scores.append(metrics.evaluate(evaluator))
+        else:
+            # logLoss metric
+            label_count = [0 for _ in range(num_models)]
+            log_loss = [0.0 for _ in range(num_models)]
+            for row in rows:
+                label_count[row.model_index] += row.total
+                log_loss[row.model_index] += row.log_loss
```
Comment (wbo4958) [inline python/src/spark_rapids_ml/classification.py:613]: Apologies for my math. So we directly add up the partial log_loss, then get the final log_loss?

**Review by wbo4958 — COMMENTED** [review]
LGTM except for a minor question.

**Review by wbo4958 — APPROVED** [review]

File: python/src/spark_rapids_ml/classification.py (hunk 2 above)
Comment (eordentlich) [reply python/src/spark_rapids_ml/classification.py:613]: that's correct.   And then divide by total number of examples to get the average log loss.

File: python/src/spark_rapids_ml/metrics/__init__.py (hunk 1 above)
Comment (eordentlich) [reply python/src/spark_rapids_ml/metrics/__init__.py:30]: Same.  Just came across it.
